api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -1060,6 +1062,17 @@ func validateNsConfUpdate(
 				)
 			}
 
+			// Add oldSingleConf ns device list into deviceList if not already present
```

Comment:
You should simplify comment, something like // keep track of devices referenced in existing namespaces

Reply:
Done